packages/adp-tooling/src/types.ts

```diff
@@ -533,6 +533,24 @@ export interface CustomConfig {
         support: ToolsSupport;
     };
 }
+export interface CloudTarget {
+    url: string | undefined;
+    authenticationType: string | undefined;
+    ignoreCertErrors: boolean;
+}
+
+export interface OnPremTarget {
+    destination?: string;
+}
+
+export type CloudCustomTaskConfigTarget = OnPremTarget | CloudTarget;
```

Comment:
I am not sure why we need these if we already have interfaces in https://github.com/SAP/open-ux-tools/blob/main/packages/system-access/src/types.ts?